er_wheels/src/er_motor_controller_node.cpp

```diff
@@ -0,0 +1,112 @@
+#include "ros/ros.h"
+#include "std_msgs/Float64.h"
+#include "std_msgs/Float32.h"
+#include "phidgets/motor_encoder.h"
+
+#include <sstream>
+#include <cmath>
+
+const int controller_frequency = 25;
+
+const double ticks_per_rev = 3591.84/4;
+
+double omega = 0;
+
+double omega_goal = 0;
+
+// use PI controller, since 25hz is probabaly to low to use d controller in a meaningful way
+double p = 0;
```

Comment:
why is this 0? we will be getting error*0 -> error won't matter at all!

Reply:
Right now I only works pretty well when driving around.
We can tune that later though or we do a feed forward controller if we have time. We can discuss that when we meet.